apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -142,33 +150,36 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
           configs: {
             numPeers: parseInt(numPeers, 10),
             meta: { federation_name: federationName },
-            modules: {
-              // TODO: figure out way to not hard-code modules here
-              1: [
-                'mint',
-                { consensus: { mint_amounts: mintAmounts }, local: {} },
-              ],
-              2: [
-                'wallet',
-                {
-                  consensus: {
-                    finality_delay: parseInt(blockConfirmations, 10),
-                    network: network as Network,
-                    client_default_bitcoin_rpc: clientDefaultBitcoinRpc,
+            modules: Object.assign(
+              {
+                // TODO: figure out way to not hard-code modules here
+                1: [
+                  'mint',
+                  { consensus: { mint_amounts: mintAmounts }, local: {} },
+                ],
+                2: [
+                  'wallet',
+                  {
+                    consensus: {
+                      finality_delay: parseInt(blockConfirmations, 10),
+                      network: network as Network,
+                      client_default_bitcoin_rpc: clientDefaultBitcoinRpc,
+                    },
+                    local: {
+                      bitcoin_rpc: bitcoinRpc,
+                    },
                   },
-                  local: {
-                    bitcoin_rpc: bitcoinRpc,
+                ],
+                3: [
+                  'ln',
+                  {
+                    consensus: { network: network as Network },
+                    local: { bitcoin_rpc: bitcoinRpc },
                   },
-                },
-              ],
-              3: [
-                'ln',
-                {
-                  consensus: { network: network as Network },
-                  local: { bitcoin_rpc: bitcoinRpc },
-                },
-              ],
-            },
+                ],
+              },
+              otherModuleParams as Record<number, AnyModuleParams> // FIXME
```

Comment:
This is a hack

Reply:
I don't think this fully soles the issue because `Object.assign` is not a deep assignment, meaning if `otherModuleParams` included a module param like `client_default_bitcoin_rpc` where we were defining some of the wallet params, but not all of them, it would get blown out. I think we need to do a deep merge of these objects instead.